src/db12/__main__.py

```diff
@@ -0,0 +1,149 @@
+
+""" DIRAC Benchmark 2012 by Ricardo Graciani, and wrapper functions to
+    run multiple copies in parallel by Andrew McNab.
+    This file (dirac_benchmark.py) is intended to be the ultimate upstream
+    shared by different users of the DIRAC Benchmark 2012 (DB12). The
+    canonical version can be found at https://github.com/DIRACGrid/DB12
+    This script can either be imported or run from the command line:
+    ./dirac_benchmark.py NUMBER
+    where NUMBER gives the number of benchmark processes to run in parallel.
+    Run  ./dirac_benchmark.py help  to see more options.
+"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+import argparse
+import json
+from pkg_resources import get_distribution, DistributionNotFound
+
+from db12 import single_dirac_benchmark
+from db12 import multiple_dirac_benchmark
+from db12 import wholenode_dirac_benchmark
+from db12 import jobslot_dirac_benchmark
+
+try:
+    VERSION = get_distribution("db12").version
+except DistributionNotFound:
+    pass
+
+def dump_as_json(filename, output):
+    """Function to save result to a json file"""
+    with open(filename, 'w') as outfile:
+        json.dump(output, outfile)
+
+def single_dirac_benchmark_cli(args):
+    '''Function that calls single_dirac_benchmark and prints
+    its results/calls dump_as_json'''
+    result = single_dirac_benchmark()["NORM"]
+    if args.json:
+        dump_as_json(args.json, result)
+    else:
+        print(result)
+
+def jobslot_dirac_benchmark_cli(args):
+    '''Function that calls jobslot_dirac_benchmark and prints
+    its results/calls dump_as_json'''
+    result = jobslot_dirac_benchmark(args.iterations, args.extra_iteration)
+    if args.json:
+        dump_as_json(args.json, result)
+    else:
+        print(
+            result["copies"], result["sum"],
+            result["arithmetic_mean"],
+            result["geometric_mean"],
+            result["median"],
+        )
+        print(" ".join([str(j) for j in result["raw"]]))
+
+def multiple_dirac_benchmark_cli(args):
+    '''Function that calls multiple_dirac_benchmark and prints
+    its results/calls dump_as_json'''
+    result = multiple_dirac_benchmark(int(args.copy), args.iterations, args.extra_iteration)
```

Comment:
I guess you do not need to convert `args.copy` here since you specified `type=int` in the subparser. 
```suggestion
    result = multiple_dirac_benchmark(args.copy, args.iterations, args.extra_iteration)
```